optuna/testing/objectives.py

```diff
@@ -0,0 +1,10 @@
+from optuna import TrialPruned
+from optuna.trial import Trial
+
+
+def fail_objective(_: Trial) -> float:
+    raise ValueError
+
+
+def pruned_objective(trial: Trial) -> float:
+    raise TrialPruned()
```

Comment:
`pruned_objective` is only used in `tests/test_callbacks.py`, and I don't think we need to extract it.

Reply:
Sorry, I forgot to replace four more places. PTAL.

Reply:
Thanks!